src/index.html

```diff
@@ -1,14 +1,74 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible"
+      content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+      href="./style.css">
+    <link rel="preconnect"
+      href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo__link">
+        <a class="logo__img"
+          href="#">
+          <img src="./images/logo.png"
+            alt="Moyo logo">
+        </a>
```

Comment:
[Checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md):
- Keep your attributes correctly formatted
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/c99c0dec-e69c-4ed7-9cdd-efed6bb46e9c)

Fix in all places

Reply:
Not fixed